generators/server/templates/src/main/kotlin/package/client/TokenRelayRequestInterceptor.kt.ejs

```diff
@@ -27,7 +27,7 @@ class TokenRelayRequestInterceptor(private val authorizationHeaderUtil: Authoriz
 
     override fun apply(template: RequestTemplate) {
         val authorizationHeader = authorizationHeaderUtil.getAuthorizationHeader()
-        authorizationHeader?.run { template.header(AUTHORIZATION, this) }
+        authorizationHeader.ifPresent { template.header(AUTHORIZATION, it) }
```

Comment:
This could be avoided if return type of AuthorizationHeaderUtil.getAuthorizationHeader() is changed to String? instead of Optional<String>. See also comment in AuthorizationHeaderUtil